### PR TITLE
Removing extra printer methods.

### PR DESCRIPTION
**Pull request by hsorby**
Addresses issue #332.

### CODE REVIEW

File: src/api/libcellml/printer.h [hunk 1]
```diff
@@ -47,7 +47,7 @@ class LIBCELLML_EXPORT Printer: public Logger
      *
      * @return The @c std::string representation of the @c Model.
      */
-    std::string printModel(const ModelPtr &model) const;
+    std::string printModel(ModelPtr smodel) const;
```
Comment (agarny) [inline src/api/libcellml/printer.h]: I believe this will break Clang-Tidy...

**Comment by hsorby** [comment]
This should be good to merge now.

File: tests/component/component.cpp [hunk 2]
```diff
@@ -18,47 +18,68 @@ limitations under the License.
 
 #include <libcellml>
 
+#include "test_utils.h"
```
Comment (agarny) [inline tests/component/component.cpp]: For consistency, this header should be listed before any other header.

File: tests/coverage/coverage.cpp [hunk 3]
```diff
@@ -65,9 +65,7 @@ TEST(Coverage, units)
     // Copy constructor
     libcellml::Units uc(um);
 
-    libcellml::Printer printer;
-    const std::string a = printer.printUnits(uc);
-    EXPECT_EQ(e, a);
+    EXPECT_EQ("dimensionless", uc.getName());
```
Comment (agarny) [inline tests/coverage/coverage.cpp:68]: `"dimensionless"` really ought to be declared as a string variable. (Same in `unitsGetVariations

File: tests/coverage/coverage.cpp [hunk 4]
```diff
@@ -80,16 +78,15 @@ TEST(Coverage, when)
     libcellml::When wm;
     libcellml::Reset r;
 
+    w.setValue("4738");
```
Comment (agarny) [inline tests/coverage/coverage.cpp]: `"4738"` ought to be declared as a string variable (and reused below in the `EXPECT_EQ()` statement).

File: tests/math/math.cpp [hunk 5]
```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include <libcellml>
 
+#include "test_utils.h"
```
Comment (agarny) [inline tests/math/math.cpp]: For consistency, this header should be listed before any other header.

File: tests/printer/printer.cpp [hunk 6]
```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include <libcellml>
 
+#include "test_utils.h"
```
Comment (agarny) [inline tests/printer/printer.cpp]: For consistency, this header should be listed before any other header.

File: tests/reset/reset.cpp [hunk 7]
```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include <libcellml>
 
+#include "test_utils.h"
```
Comment (agarny) [inline tests/reset/reset.cpp]: For consistency, this header should be listed before any other header.

**Comment by agarny** [comment]
It looks fine to me, pending some minor suggestions.

**Review by agarny — APPROVED** [review]

**Review by nickerso — APPROVED** [review]

**Pull request merged**